### DISCUSSION TITLE
IMPRESSORA COMEÇA IMPRESSÃO NO CANTO DA MESA

### DISCUSSION THREAD

**Discussion by mrcharlestom** [mriscoc/Ender3V2S1#1245] [Q&A]
Tenho uma ENDER 3 V2 NEO, funcionamento perfeito, nunca tive problemas, porém resolvi atualizar o firmware dela para obter mais opções de configurações e ter o sistema atualizado, enfim, fiz todo o processo de atualização e deu tudo certo.
Utilizei o firmware desse link ( https://github.com/mriscoc/Ender3V2S1/releases/download/20240122/Ender3V2-422-BLTUBL-MPC-20240125.bin ), que é o que dá certo com minha placa, segui todos os passos conforme mostrado nos vídeos de instalação e configuração.
Quando inicio uma impressão, ela funciona tudo normal, busca o home, inicia a impressão das linhas laterais para limpeza do bico, apenas quando inicia a impressão do projeto mesmo, onde ela teria de começar no centro da mesa, ela inicia bem no canto da mesa.
Já tentei outro firmware diferente e deu mesmo problema.
Utilizo o Cura para fatiar as peças e nunca tive problemas!
![WhatsApp Image 2024-02-02 at 20 06 34](https://github.com/mriscoc/Ender3V2S1/assets/158598009/e636b5a6-6b8d-4db8-bca0-d676b2e58d95)

Alguém pode me ajudar por favor?

**Comment by mriscoc** [comment]
That could be a problem with the printer profile in the slicer, delete it and create a new one.

**Comment by araoludiosa** [comment]
Boa noite, estou tendo o mesmo problema, porem a minha impressora estava perfeita e começou a fazer isso do nada.
exatamente como a sua, começa a imprimir o canto frontal esquerdo e até mesmo fora da mesa.
no meu caso eu resolvi tentar atualizar o firm para resolver isso, porem nao resolveu,
sei que nao é problema de fateamento.
você ja conseguiu resolver a sua ?

**Reply by mrcharlestom** [reply]
confere o tamanho da mesa no fatiador que você usa, deve estar maior do que sua impressora, se estiver correto, abra seu firmware com um programa editor e procure pelo tamanho da mesa nele também, deve estar maior que sua impressora também, um desses dois deve estar errado!

**Reply by araoludiosa** [reply]
Bom dia ! Obrigado pela dica, mas no fateador esta duto certo, inclusive estou tentando imprimir arquivos que ja havia impresso antes com sucesso, por isso acredito que não haja erro no fateador.
quando ao firmware, ja tentei diversas firmware diferentes mas sempre com o mesmo problema.
mesmo asssim, ainda poderia testar essa questao de editar o firmware porem nao tenho conhecimento para isso, nem saberia qual programa usar.

**Reply by mrcharlestom** [reply]
eu resolvi meu problema alterando o tamanho da mesa, também achei que estava certo mas quando fui conferir estava errado, dá uma olhada nesse vídeo, deve te ajudar:
https://www.youtube.com/watch?v=DJXkLqvaNFY

**Comment by mriscoc** [comment]
Follow the calibration guide: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
